src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -864,7 +864,7 @@ static void Main()
         {
             throw new System.Exception();
         }
-        catch (System.Exception) when (false)
+        catch (System.Exception) when (default)
```

Comment:
The test class is TargetTypedDefaultTest so I am assuming it is going to contain 'default' related tests. Other cases will be covered by the other set of additional tests that I've written, right? Or should I repeat them here as well as a pair?